analyses/driver/matcher.go

```diff
@@ -9,19 +11,64 @@ type RequestType struct {
 	DepsTreePath    string
 }
 
+// CIFormat is SA Result output format required by CI
+type CIFormat struct {
+	TotalScannedDependencies                   int          `json:"total_scanned_dependencies"`
+	DirectDependenciesWithKnownVulnerabilities int          `json:"direct_dependencies_with_known_vulnerabilites"`
+	DirectDependenciesWithSynkAdvisories       int          `json:"direct_dependencies_with_synk_advisories"`
+	VulnerabilitiesPerSeverity                 SeverityType `json:"vulnerabilities_per_severity"`
+}
```

Comment:
This can go into separate file. Also name seem to be weird., how about ConciseReport or StackSummary?

Reply:
Ack.